### DISCUSSION TITLE
WebGUI on x86_64 architecture.

### DISCUSSION THREAD

**Discussion by jacter74** [jvde-github/AIS-catcher#337] [Q&A] [answered]
WebGUI on x86_64 architecture.

I tried to install WebGUI using the script:
`sudo bash -c "$(curl -fsSL https://raw.githubusercontent.com/jvde-github/AIS-catcher-control/main/install_ais_catcher_control.sh)"`

Unfortunately, it is not possible. I get the message:
```
========================================
Checking and installing dependencies...
========================================
========================================
Unsupported architecture: x86_64
========================================
```
Is WebGUI really so dependent on the processor architecture or is it some kind of bug?

**Comment by jvde-github** [comment]
Well, I would say, it was never implemented as I intended this for the Raspberry devices. But included it now. Can you run the script again?

**Reply by jacter74** [reply, marked as answer]
Now everything works perfectly, exactly as you described in the documentation.

Thanks for the quick response. You rock!

**Comment by jacter74** [comment]
BTW: Is there any way to go back to being able to edit all options via WebGUI after the configuration files have been edited manually? For example, by deleting them or clearing their contents?

**Reply by jvde-github** [reply]
yes, delete your configuration files:

sudo rm /etc/AIS-catcher/*

And then create them again by calling the two install scripts, this will create fresh config files.
